client/state/timezones/actions.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+
+export const timezonesRequestAction = () => ( {
```

Comment:
Why do we suffix these action creators with "Action"? Could we make them more human-readable as we've done with other action creators, e.g. `requestTimezones`, `receiveTimezones`, etc.